tools/labs/Makefile

```diff
@@ -36,7 +36,7 @@ docker-docs:
 	cd docker && UID=$(shell id -u) GID=$(shell id -g) docker-compose run docs-build bash -c "cd /linux/tools/labs && make docs"
 
```

Comment:
@Sergiu121 please add a small commit message that tells why this change is needed.